custom_components/tahoma/const.py

```diff
@@ -76,7 +76,6 @@
     "HitachiAirToAirHeatPump": CLIMATE,  # widgetName, uiClass is HitachiHeatingSystem (not supported)
     "HumiditySensor": SENSOR,
     "Light": LIGHT,
-    "LightSensor": SENSOR,
```

Comment:
@tetienne should we already remove those sensors? Or keep this PR for secondary sensors and in a follow up rewrite our full sensor (and eventually binary sensor) system?

Having our sensor entities different will have other complications if we don't have a 'main' device/entity anymore.

Reply:
So should I remove also my StateDescription related to the light?